src/App.jsx

```diff
@@ -4,18 +4,29 @@ import { BrowserRouter, Route, Routes } from "react-router-dom";
 import { ReviewsList } from "./components/ReviewsList";
 import { SingleReview } from "./components/SingleReview";
 import { NavBar } from "./components/NavBar";
+import { SignIn } from "./components/SignIn";
+import { useState, createContext } from "react";
+import { Categories } from "./components/Categories";
+
+const UserContext = createContext();
 
 function App() {
+  const [user, setUser] = useState("");
```

Comment:
I like that you've set the initial user to be empty, which forces the user to sign in before posting a comment, well done